.github/workflows/tests.yml

```diff
@@ -43,11 +43,34 @@ jobs:
     strategy:
       matrix:
         distro:
-          - debian11
-          - debian10
+          - debian11 # Bullseye
+          - debian10 # Buster
+          - debian9 # Stretch
+          - debian8 # Jessie
+          - ubuntu2204 # Jammy
           # https://github.com/geerlingguy/docker-ubuntu2204-ansible/issues/6
-          # - ubuntu2204
-          - ubuntu2004
+          - ubuntu2004 # Focal
+          - ubuntu1804 # Bionic
```

Comment:
I have no intention of supporting Ubuntu versions that are EOL (older than 2004). Same for all those version of Fedora besides 37 and 38. And CentOS 8. There are limits of resources a repo can use on GA and I don't plan to go close to them nor wait hours to merge simple changes in the repo, there is no benefit for this much overhead. This is also a practice you can see in almost all Ansible repos.

Reply:
> @Commandcracker there are some last linting errors.. also, I think you will need to revert Ubuntu 22.04 because of a bug that I had commented in the issue you removed.

the comment is still there, ill investigate it.